app/assets/styles/_map.scss

```diff
@@ -47,6 +47,18 @@
   stroke: rgba($base-color, 0.08);
   stroke-width: 1;
   @include transition(all 0.2s ease 0s);
+
+  // Style cell shading according to layer
+  .base-layer--oam-base & {
+    fill: $primary-color;
+  }
+  .base-layer--osm & {
+    fill: $primary-color;
+  }
+  .base-layer--satellite & {
+    fill: #FFF;
+  }
```

Comment:
Define here the different fills. For now I just used the white for the satellite layer